src/pages/instances/InstanceList.tsx

```diff
@@ -316,7 +418,8 @@ const InstanceList: FC = () => {
     pagination.currentPage,
   ]);
 
-  const hasInstances = isLoading || instances.length > 0;
+  const hasInstances =
+    isLoading || instances.length > 0 || creationOperations.length > 0;
```

Comment:
There are some weird side effects:

1) Showing 1 out of 0 instances

![Screenshot from 2023-05-17 14:26:49](https://github.com/canonical/lxd-ui/assets/56583786/686510ce-5403-495b-a855-479f836dfd26)

2) Same as above but with wrong height

![Screenshot from 2023-05-17 14:31:41](https://github.com/canonical/lxd-ui/assets/56583786/63cce8ae-52d6-4120-9f98-312c3ebddaa4)